packages/mu-router/index.js

```diff
@@ -184,7 +183,7 @@ module.exports = function (opts) {
     // this will be the last step in the distributed call chain. Otherwise the message is being routed through a transport layer
     // so call the tf and invoke the local callback once the message has been sent
     if (match.type === 'callback') {
-      match.tf(mue.remote(message.err || null), message.response)
+      match.tf(mue.remote(message.err || null), message.response, message)
```

Comment:
if we go with what I outlined in https://github.com/apparatus/mu/issues/71#issuecomment-265880735 and https://github.com/apparatus/mu/issues/43 we can do this without any breakage